Makefile

```diff
@@ -46,6 +48,13 @@ cargo:
 clean:
 	rm -rf target/
 
+clientgen:
+	docker run --rm -v "$(shell pwd):/local" --user "$(shell id -u):$(shell id -g)" openapitools/openapi-generator-cli generate \
+		-i /local/openapi.yml \
+		-g rust \
+		-o /local/client
```

Comment:
Probably want to add something like:
`--additional-properties=packageName=cloudtruth-openapi,packageVersion=0.6.0,supportAsync=false` to the code generation to tweak a few things.

Reply:
Thanks, that changed the package name and version so it was more useful.

Here are the errors though, there might be open issues in openapi-generator for these, not sure.

```
   Compiling cloudtruth-openapi v0.6.0 (/home/dev/cli/client)
error[E0412]: cannot find type `OneOfRoleEnumBlankEnum` in module `crate::models`
  --> src/models/user_create.rs:28:41
   |
28 |     pub role: Option<Box<crate::models::OneOfRoleEnumBlankEnum>>,
   |                                         ^^^^^^^^^^^^^^^^^^^^^^ not found in `crate::models`

error[E0282]: type annotations needed
  --> src/models/user_create.rs:40:19
   |
40 |             role: None,
   |                   ^^^^ cannot infer type for type parameter `T` declared on the enum `Option`

error: aborting due to 2 previous errors
```